frontend/scripts/react-components/nav/filters-nav/resize-by-selector/resize-by-selector.container.js

```diff
@@ -1,13 +1,13 @@
 import { connect } from 'react-redux';
 import { toggleDropdown } from 'actions/app.actions';
 import { selectResizeBy } from 'actions/tool.actions';
-import ResizeBy from 'react-components/tool/nav/resize-by.component';
+import ResizeBySelector from './resize-by-selector.component';
```

Comment:
I personally avoid relative imports as much as possible - makes it easier to move code around, and to get context on the things being imported. What do you think?

Reply:
Granted